index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Task 1</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="body">
+        <div class="nav">
+            <ul class="navbar-nav">
+                <li class="nav-item">
+                    <a href="#" class="icon1 nav-link"></a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="icon2 nav-link"></a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="icon3 nav-link"></a>
+                </li>
+            </ul>
+            <ul class="navbar-nav2">
+                <li class="nav-item">
+                    <a href="#" class="icon4"></a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="icon5"></a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="icon6"></a>
+                </li>
+            </ul>
+        </div>
+        <div class="content">
+            <div class="plans">
+                <div class="plan">
+                    <div class="plan_data">
+                        <div class="uper">
+                            <div class="type_a type">
+                                <p>
+                                    Basic Plan
+                                </p>
+                            </div>
+                            <h2>
+                                2500 Credits
+                            </h2>
+                            <h1>
+                                $1250
+                            </h1>
+                            <p>
+                                kick start your career with this pack
+                            </p>
+                            <button>
+                                Buy this bundle
+                            </button>
+                        </div>
+                        <div class="info">
+                            <ul style="list-style-type: circle;">
+                                <div class="list"><div class="tick"><img src="Icon.svg" alt="svg image"></div>Per credit : <span>&nbsp; $0.5</span></div>
```

Comment:
extra should be div remove img tag like this,
`<img class="tick" src="Icon.svg" alt="svg image">`